include/swift/Sema/CSBindings.h

```diff
@@ -469,8 +469,11 @@ class BindingSet {
   ASTNode getAssociatedCodeCompletionToken() const {
     return Info.AssociatedCodeCompletionToken;
   }
+  
+  SmallVector<KnownProtocolKind> getLiteralRequirements() const;
```

Comment:
Sorry I forgot to leave a note about this. Let's change this to `void getLiteralRequirements(SmallVectorImpl<KnownProtocolKind> &requirements) const;` because `SmallVector` without initial capacity would create problems on non-macOS platforms, and it's better to pass it in instead of returning.